internal/proto/controller/api/resources/authmethods/v1/auth_method.proto

```diff
@@ -101,7 +101,7 @@ message PasswordAuthMethodAttributes {
       this: "attributes.min_login_name_length"
       that: "MinLoginNameLength"
     }
-  ]; // @gotags: `class:"public"`
+  ]; // @gotags: `class:"public" eventstream:"observation"`
```

Comment:
@jimlambrt I was unsure about these 2 fields to see if we should include them or not. `min_login_name_length` & `min_password_length`

Reply:
I would say: nope.  It's customer input.